api/client/trust.go

```diff
@@ -425,6 +425,10 @@ func (cli *DockerCli) trustedPush(repoInfo *registry.RepositoryInfo, tag string,
 	}
 
 	err = repo.Publish()
+	if err == nil {
+		fmt.Fprintf(cli.out, "Finished signing %q with tag %s\n", repoInfo.FullName(), tag)
```

Comment:
@aaronlehmann I don't want to support the "push all" tags behavior by accident. We want trust signatures to always take an explicit tag.

We should remove the code that attempts to do smart logic around signing multiple tags.